features_module_how_to_share_sites_config_with_oth.md

```diff
@@ -0,0 +1,130 @@
+
+When you are done, just click on `Download feature`.
+Uncompress the file, and put the module folder under `modules/custom` in the project directory.
+Enable the module so Drupal can start using it.
+
+**Note**: When you make your configurations in DB, then export it as a Feature and enable the module, your configurations are stored both in the code and DB. In this case, you don't know if what you are seeing on the screen is coming from the code or from the DB, so you can't be sure if the module is working as expected. The most effective way to make sure all our configuration moved successfully to the code, is to start a new installation and then enable the module. Now, after a new installation, you know that everything you see is from the code.
+
+
+## Changing a Feature
+
+Now we have our Blog Feature (module) installed and running on our site. But what if we want to make changes in our configuration? For example, change the page title to be 'Gizra blog' instead of 'Blog'?
+We can simply go to the View configuration in the UI and change the title.
+
+![](images/features/change_title.png)
+
+But where is this change stored? Right, it is stored in DB. So now we have the title 'Gizra blog' in DB and the title 'Blog' in code.
+
+Features module noticed this change made to the component and marks it with an `Overridden` state on the Features administration page.
+
+![](images/features/overridden.png)
+
+Overridden state means that the configuration in DB is different from configuration in code, and what stored in DB is stronger, so it overrides the code.
+This is a situation we need to solve. We can choose to revert to the original code or to update the Feature by applying the new changes to it. 
+
+
+## Reverting a Feature
+
+Click on the `Overridden` signal, then you will see the component that was override (View in this case because we changed the View title). Check the component you want to revert and click on `Revert components`
+
+![](images/features/revert.png)
+
+
+Now go back to Features administration page and you will see that the state is `Default`, means the feature configurations in code are the same like in DB.
+
+![](images/features/default_state.png)
+
+
+## Updating a Feature
+
+When we make changes to the configuration and want to keep them, we need to update the code.
+One way to do this is through the UI: click on `recreate` and download the Feature again. Then replace the old Feature folder in module directory with the new one.
+A second and much quicker way is to use Drush. Run `drush fu [feature name]` in Drupal directory (www) and that's it, your Feature is updated!
+
+If you go to Features administration page, you will see that the state is `Default` again.
+
+
+## What to “Featurize”? 
+
+In short: everything that is _not_ content!
+
+Everything we have in our local DB should be in code, so we can share it.
+
+Of course we can't Featurize content. Content needs to be [migrated](https://www.drupal.org/project/migrate).
```

Comment:
I think it's better to link to one of Gizra's post about migrate, rather than just the module.